### PR TITLE
add  sostrades-radouanbouadel

### PR DESCRIPTION
**Pull request by sostrades-radouanbouadel**

### CODE REVIEW

**Comment by durandom** [comment]
@sostrades-radouanbouadel can you have a look at my comment and run the pre-commit to fix the whitespace issue?
See https://github.com/operate-first/apps/blob/master/contributing.md#linting

File: github-config.yaml [hunk 1]
```diff
@@ -334,7 +336,19 @@ orgs:
         description: "Repository of ArgoCD application for workshop purposes "
         has_projects: false
         has_wiki: false
+      sostrades:
```
[the remark this answers is not in the document]
Comment (sostrades-radouanbouadel) [reply github-config.yaml:339]: I need to login to Argocd, dont need any  repo neither in os-climate org nor in OPF org, just find the solution to gt access to Argocd

[the remark this answers is not in the document]
Comment (redmikhail) [reply github-config.yaml:339]: in this case you can remove repo and team definition . Ultimately you just need to be member in the org , comparing to the version of the file in main branch you need to have following entries under members:
`    - sostrades-radouanbouadel
      - sostrades-dpeltre`
make sure that there is no trailing spaces after names of the members

**Comment by redmikhail** [comment]
@sostrades-radouanbouadel access that you were looking for was facilitated through separate PR https://github.com/operate-first/common/pull/162 . If it covers all what you were looking for , please close this PR

**Comment by redmikhail** [comment]
Closing this PR as there are no activity and requirements were addressed by other PR

**Pull request closed without merging**